model/host/db.go

```diff
@@ -427,3 +427,40 @@ func GetHostsByFromIdWithStatus(id, status, user string, limit, sortDir int) ([]
 	}
 	return hosts, nil
 }
+
+// hostStatsByDistroPipeline returns a pipeline that will group all up hosts by distro
+// and return the count of hosts as well as how many are running tasks
+func hostStatsByDistroPipeline() []bson.M {
+	return []bson.M{
+		bson.M{
```

Comment:
cool trick, you can omit type names in slice elements, because the compiler can add them in, so you can just do: 

`[]bson.M{ {}, {}, {} }`